tools/config/go-templates/api.mustache

```diff
@@ -154,7 +154,7 @@ func (a *{{{classname}}}Service) {{{nickname}}}Execute(r {{operationId}}ApiReque
 	}
 
 	localVarPath := localBasePath + "{{{path}}}"{{#pathParams}}
-	localVarPath = strings.Replace(localVarPath, "{"+"{{baseName}}"+"}", url.PathEscape(parameterValueToString(r.{{paramName}}, "{{paramName}}")), -1){{/pathParams}}
+	localVarPath = strings.Replace(localVarPath, "{"+"{{baseName}}"+"}", url.PathEscape(r.{{paramName}}), -1){{/pathParams}}
```

Comment:
I assume that you have tested all the type parameters we have in the API

Reply:
all TF tests passed and they have a good coverage.
also SDK wouldn't compile if some param were not a string, as it's required by strings.Replace

Reply:
That is odd. We do have integer and time params in API. It is just that in this structure they always represented as strings?  We definitely do have time based params. Very clever to use TF tests here

Reply:
note that this function is called only in path params, not in the body

Reply:
Yes. Upstream SDK supports **Object** based parameters but they are rare in openapi and do not exist in Atlas.